molo/profiles/tests/test_views.py

```diff
@@ -1546,7 +1546,8 @@ def test_incorrect_security_answer_gets_error(self):
         self.failUnless(error_message in response.content)
 
     def test_too_many_retries_result_in_error(self):
-        error_message = "Too many attempts"
```

Comment:
This change in error message seems to obscure the reason that the error is being thrown.